Crash cause of invalid yaml?

**Describe the bug**
Added resources to the services, okteto up crashes now

**To Reproduce**
```yaml
name: shopware
image: shopware/development
command: ["bash"]
mountpath: "/app"
environment:
    - WEB_DOCUMENT_ROOT=/app/public
forward:
    - 8080:80
reverse:
    - 9000:9000
resources:
    limits:
        memory: "2Gi"
        cpu: "1000m"
services:
    - name: mysql
      image: mysql:5.7
    - name: nginx
      mountpath: "/var/www/html"
      resources:
          limits:
              memory: "2Gi"
              cpu: "2000m"
      environment:
          - HTTPS=on
```
Error
```
panic: assignment to entry in nil map [recovered]
        panic: assignment to entry in nil map

goroutine 1 [running]:
gopkg.in/yaml%2ev2.handleErr(0xc0005cba80)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/yaml.go:249 +0xa1
panic(0x1bff4e0, 0x237ddb0)
        /usr/local/go/src/runtime/panic.go:679 +0x1c0
github.com/okteto/okteto/pkg/model.(*ResourceList).UnmarshalYAML(0xc0004cebb8, 0xc000fe9ca0, 0xc000028000, 0x1c26440)
        /home/circleci/project/pkg/model/serializer.go:155 +0x191
gopkg.in/yaml%2ev2.(*decoder).callUnmarshaler(0xc000fdc180, 0xc000421e30, 0x83901e0, 0xc0004cebb8, 0xc0004cebb8)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:270 +0xb3
gopkg.in/yaml%2ev2.(*decoder).prepare(0xc000fdc180, 0xc000421e30, 0x1c36300, 0xc0004cebb8, 0x195, 0x6, 0xc001069501, 0x198, 0x0)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:313 +0x19e
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc000fdc180, 0xc000421e30, 0x1c36300, 0xc0004cebb8, 0x195, 0xc0004cebb8)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:360 +0xea
gopkg.in/yaml%2ev2.(*decoder).mappingStruct(0xc000fdc180, 0xc000421d50, 0x1c95420, 0xc0004cebb8, 0x199, 0x0)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:763 +0xa73
gopkg.in/yaml%2ev2.(*decoder).mapping(0xc000fdc180, 0xc000421d50, 0x1c95420, 0xc0004cebb8, 0x199, 0x1c95420)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:622 +0xa9b
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc000fdc180, 0xc000421d50, 0x1c95420, 0xc0004cebb8, 0x199, 0xc0004cebb8)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:368 +0x1b6
gopkg.in/yaml%2ev2.(*decoder).mappingStruct(0xc000fdc180, 0xc000421ab0, 0x1e50f60, 0xc0004cea80, 0x199, 0xc0005cb1f0)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:763 +0xa73
gopkg.in/yaml%2ev2.(*decoder).mapping(0xc000fdc180, 0xc000421ab0, 0x1e50f60, 0xc0004cea80, 0x199, 0x1e50f60)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:622 +0xa9b
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc000fdc180, 0xc000421ab0, 0x1dcdc80, 0xc0004ee268, 0x196, 0x196)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:368 +0x1b6
gopkg.in/yaml%2ev2.(*decoder).sequence(0xc000fdc180, 0xc000421730, 0x1afa7e0, 0xc0004ce8e8, 0x197, 0x1afa7e0)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:605 +0x263
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc000fdc180, 0xc000421730, 0x1afa7e0, 0xc0004ce8e8, 0x197, 0xc0004ce8e8)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:370 +0x18a
gopkg.in/yaml%2ev2.(*decoder).mappingStruct(0xc000fdc180, 0xc000420690, 0x1e50f60, 0xc0004ce780, 0x199, 0x1a0b2e6)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:763 +0xa73
gopkg.in/yaml%2ev2.(*decoder).mapping(0xc000fdc180, 0xc000420690, 0x1e50f60, 0xc0004ce780, 0x199, 0x1e50f60)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:622 +0xa9b
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc000fdc180, 0xc000420690, 0x1e50f60, 0xc0004ce780, 0x199, 0x0)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:368 +0x1b6
gopkg.in/yaml%2ev2.(*decoder).document(0xc000fdc180, 0xc000420620, 0x1e50f60, 0xc0004ce780, 0x199, 0x9ca857)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:380 +0x83
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc000fdc180, 0xc000420620, 0x1e50f60, 0xc0004ce780, 0x199, 0x199)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/decode.go:356 +0x24e
gopkg.in/yaml%2ev2.unmarshal(0xc00106a400, 0x1da, 0x3da, 0x1dcdc80, 0xc0004ce780, 0xc001055901, 0x0, 0x0)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/yaml.go:148 +0x363
gopkg.in/yaml%2ev2.UnmarshalStrict(...)
        /go/pkg/mod/gopkg.in/yaml.v2@v2.2.4/yaml.go:89
github.com/okteto/okteto/pkg/model.Read(0xc00106a400, 0x1da, 0x3da, 0x1da, 0x3da, 0x0)
        /home/circleci/project/pkg/model/dev.go:183 +0x28a
github.com/okteto/okteto/pkg/model.Get(0x1ef5718, 0xa, 0xc001069101, 0xc00105ef28, 0xc00105ef10)
        /home/circleci/project/pkg/model/dev.go:150 +0x75
github.com/okteto/okteto/cmd.loadDev(0x1ef5718, 0xa, 0x1, 0x11cd9ce, 0xc00016e500)
        /home/circleci/project/cmd/utils.go:26 +0x194
github.com/okteto/okteto/cmd.Up.func1(0xc000fae000, 0x386b110, 0x0, 0x0, 0x0, 0x0)
        /home/circleci/project/cmd/up.go:94 +0xe9
github.com/spf13/cobra.(*Command).execute(0xc000fae000, 0x386b110, 0x0, 0x0, 0xc000fae000, 0x386b110)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x467
github.com/spf13/cobra.(*Command).ExecuteC(0xc000fa6280, 0xc0005cbf30, 0x1, 0x1)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x302
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
        /home/circleci/project/main.go:69 +0x6b8
```


**Expected behavior**
It works

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version newest

**Additional context**

--- a/pkg/model/dev.go
+++ b/pkg/model/dev.go
@@ -177,11 +177,7 @@ func Read(bytes []byte) (*Dev, error) {
 		Command:     make([]string, 0),
 		Forward:     make([]Forward, 0),
 		Volumes:     make([]Volume, 0),
-		Resources: ResourceRequirements{
-			Limits:   ResourceList{},
-			Requests: ResourceList{},
-		},
-		Services: make([]*Dev, 0),
+		Services:    make([]*Dev, 0),
 	}
 
 	if err := yaml.UnmarshalStrict(bytes, dev); err != nil {

--- a/pkg/model/dev_test.go
+++ b/pkg/model/dev_test.go
@@ -34,50 +34,77 @@ securityContext:
     - SYS_TRACE
     drop:
     - SYS_NICE
-workdir: /app`)
-	d, err := Read(manifest)
+workdir: /app
+services:
+  - name: deployment
+    container: core
+    image: code/core:0.1.8
+    command: ["uwsgi"]
+    annotations:
+      key1: value1
+      key2: value2
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+    securityContext:
+      capabilities:
+        add:
+          - SYS_TRACE
+        drop:
+          - SYS_NICE
+    workdir: /app`)
+	main, err := Read(manifest)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	if d.Name != "deployment" {
-		t.Errorf("name was not parsed: %+v", d)
+	if len(main.Services) != 1 {
+		t.Errorf("'services' was not parsed: %+v", main)
 	}
+	for _, dev := range []*Dev{main, main.Services[0]} {
+		if dev.Name != "deployment" {
+			t.Errorf("'name' was not parsed: %+v", main)
+		}
 
-	if len(d.Command) != 1 || d.Command[0] != "uwsgi" {
-		t.Errorf("command was not parsed: %+v", d)
-	}
+		if len(dev.Command) != 1 || dev.Command[0] != "uwsgi" {
+			t.Errorf("command was not parsed: %+v", dev)
+		}
 
-	memory := d.Resources.Requests["memory"]
-	if memory.String() != "64Mi" {
-		t.Errorf("Resources.Requests.Memory was not parsed: %s", memory.String())
-	}
+		memory := dev.Resources.Requests["memory"]
+		if memory.String() != "64Mi" {
+			t.Errorf("Resources.Requests.Memory was not parsed: %s", memory.String())
+		}
 
-	cpu := d.Resources.Requests["cpu"]
-	if cpu.String() != "250m" {
-		t.Errorf("Resources.Requests.CPU was not parsed correctly. Expected '250M', got '%s'", cpu.String())
-	}
+		cpu := dev.Resources.Requests["cpu"]
+		if cpu.String() != "250m" {
+			t.Errorf("Resources.Requests.CPU was not parsed correctly. Expected '250M', got '%s'", cpu.String())
+		}
 
-	memory = d.Resources.Limits["memory"]
-	if memory.String() != "128Mi" {
-		t.Errorf("Resources.Requests.Memory was not parsed: %s", memory.String())
-	}
+		memory = dev.Resources.Limits["memory"]
+		if memory.String() != "128Mi" {
+			t.Errorf("Resources.Requests.Memory was not parsed: %s", memory.String())
+		}
 
-	cpu = d.Resources.Limits["cpu"]
-	if cpu.String() != "500m" {
-		t.Errorf("Resources.Requests.CPU was not parsed correctly. Expected '500M', got '%s'", cpu.String())
-	}
+		cpu = dev.Resources.Limits["cpu"]
+		if cpu.String() != "500m" {
+			t.Errorf("Resources.Requests.CPU was not parsed correctly. Expected '500M', got '%s'", cpu.String())
+		}
 
-	if d.Annotations["key1"] != "value1" && d.Annotations["key2"] != "value2" {
-		t.Errorf("Annotations were not parsed correctly")
-	}
+		if dev.Annotations["key1"] != "value1" && dev.Annotations["key2"] != "value2" {
+			t.Errorf("Annotations were not parsed correctly")
+		}
 
-	if !reflect.DeepEqual(d.SecurityContext.Capabilities.Add, []apiv1.Capability{"SYS_TRACE"}) {
-		t.Errorf("SecurityContext.Capabilities.Add was not parsed correctly. Expected [SYS_TRACE]")
-	}
+		if !reflect.DeepEqual(dev.SecurityContext.Capabilities.Add, []apiv1.Capability{"SYS_TRACE"}) {
+			t.Errorf("SecurityContext.Capabilities.Add was not parsed correctly. Expected [SYS_TRACE]")
+		}
 
-	if !reflect.DeepEqual(d.SecurityContext.Capabilities.Drop, []apiv1.Capability{"SYS_NICE"}) {
-		t.Errorf("SecurityContext.Capabilities.Drop was not parsed correctly. Expected [SYS_NICE]")
+		if !reflect.DeepEqual(dev.SecurityContext.Capabilities.Drop, []apiv1.Capability{"SYS_NICE"}) {
+			t.Errorf("SecurityContext.Capabilities.Drop was not parsed correctly. Expected [SYS_NICE]")
+		}
 	}
 }
 

--- a/pkg/model/serializer.go
+++ b/pkg/model/serializer.go
@@ -115,6 +115,10 @@ func (r *ResourceList) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return err
 	}
 
+	if *r == nil {
+		*r = ResourceList{}
+	}
+
 	for k, v := range raw {
 		parsed, err := resource.ParseQuantity(v)
 		if err != nil {

